src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -245,6 +245,11 @@ private static byte toByte(boolean ignoreUnavailable, boolean allowNoIndices, bo
         return id;
     }
 
+    @Override
+    public String toString() {
+        return "id: " + this.id;
+    }
```

Comment:
I don't think this is needed, also not meaningful as it would print out a "random" number, it's not something that users can understand